File: swift/AntennaProtocol/Sources/AntennaProtocol/Topics.swift
import Foundation

public enum MBTopics {
    public static func categoryTopic(_ categoryEns: String) -> String {
        return "mb/v1/cat/\(categoryEns)"
    }

    public static func helpTopic(_ categoryEns: String) -> String {
        return "mb/v1/help/\(categoryEns)"
    }

    public static func helpRepliesTopic(_ helpRequestEventId: String) -> String {
        return "mb/v1/help-replies/\(helpRequestEventId)"
    }
}

File: rust/antenna-protocol/src/topics.rs
pub fn category_topic(category_ens: &str) -> String {
    format!("mb/v1/cat/{}", category_ens)
}

pub fn help_topic(category_ens: &str) -> String {
    format!("mb/v1/help/{}", category_ens)
}

pub fn help_replies_topic(help_request_event_id: &str) -> String {
    format!("mb/v1/help-replies/{}", help_request_event_id)
}
